asistente_ladm_col/gui/supplies/dlg_missing_cobol_supplies.py

```diff
@@ -63,6 +63,7 @@ def __init__(self, db, conn_manager, parent=None):
         self.txt_file_path_uni.textChanged.emit(self.txt_file_path_uni.text())
         self.txt_file_path_gdb.textChanged.emit(self.txt_file_path_gdb.text())
         self.txt_file_path_folder_supplies.textChanged.emit(self.txt_file_path_folder_supplies.text())
+        self.txt_files_name_supplies.textChanged.emit(self.txt_files_name_supplies.text())
```

Comment:
Change name to `self.txt_file_names_supplies`.

Reply:
I already changed in the last commit, thanks